src/components/ec-currency-input/ec-currency-input.vue

```diff
@@ -186,70 +186,61 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/tools/index';
-@import '../../scss/settings/index';
-
-$ec-currency-input-invalid-color: $color-error !default;
-
+<style>
 .ec-currency-input {
-  z-index: 0; // When in currencies dropdown is focused gets a high z-index. We set the index here of the whole component to 0 so we eliminate the chance of competing with other components
+  z-index: 0;
 
   &__input-group {
-    display: flex;
-    flex-direction: row;
+    @apply tw-flex tw-flex-row;
   }
 
   &__currencies {
-    $currencies-width: 104px;
+    --currencies-width: 104px;
```

Comment:
local CSS variables are not supported by polyfills: https://www.npmjs.com/package/css-vars-ponyfill (Limitations):
> Custom property declaration support is limited to :root and :host rulesets

You can create this var globally: `--ec-currency-input-currencies-width: 104px`

Reply:
Thank you Milos, I've created that variable in layout.css